recipes/llvmdev38/meta.yaml

```diff
@@ -40,7 +40,8 @@ requirements:
 test:
   commands:
     - $PREFIX/lib/llvm-{{ shortversion }}/bin/llvm-config --libs  # [not win]
-    - $PREFIX/lib/llvm-{{ shortversion }}/bin/llc -version
+    - $PREFIX/lib/llvm-{{ shortversion }}/bin/llc -version  # [not win]
+    - %PREFIX%\lib\llvm-{{ shortversion }}/bin\llc -version  # [win]
```

Comment:
I think the linter doesn't like the `\`. Also there is a weird mix of `\` and `/` here (see near `bin`). So it could be that.

Though `conda` will convert `/` to `\` on Windows. So I'd stick with that. Alternatively if we need `\` for some reason, double up on them `\\` so as to escape them to avoid getting special characters.